foundation-extensions/mobile-core/rules-engine/README.md

```diff
@@ -4,7 +4,7 @@ The Rules Engine is provided by the Adobe Experience Platform Mobile SDKs as par
 
 In the Experience Platform Mobile SDKs, the Rules Engine enables you to integrate the data and functionality of marketing and ad tech to help disparate products communicate. The Rules Engine looks for the user interaction and associated data, and when the criteria that you defined in the rules are met, the actions that you specified are triggered.
 
-The Rules Engine, with [Experience Platform Launch](https://launch.adobe.com/)'s tag management system, allows for complex if/then/else workflows that can combine multiple solution and extension related behaviors. You can define and capture the necessary user data and orchestrate how each solution contributes to the user's experience.
+The Rules Engine, with [Experience Platform Launch](https://experience.adobe.com/#/data-collection//)'s tag management system, allows for complex if/then/else workflows that can combine multiple solution and extension related behaviors. You can define and capture the necessary user data and orchestrate how each solution contributes to the user's experience.
```

Comment:
is the extra `/` at the end of `https://experience.adobe.com/#/data-collection//` needed?

Reply:
Nope! I just did a replace all for launch.adobe.com (since that's no longer the correct URL), but didn't include the trailing slash. I'll go fix that (and other instances where I forgot the trailing slash) as well!